### ISSUE TITLE
Batch script, group by base not working as expected

### ISSUE THREAD

**Issue by BuysDB** [igvteam/igv#796]
I am experiencing issues using the batch script mode.
A script like:
```
goto chr1:123456
group base_at_pos
```
Results in grouping at the last coordinate used for grouping by hand, not the coordinate set by `goto` command. Sorting by base does run at the supplied coordinate as expected.
Should the desired coordinate be supplied in another way or is this a bug?

platform: Linux, nightly build and current stable version both show the same behavior

**Comment by jrobinso** [comment]
I don't see the position specified in your example.

**Comment by BuysDB** [comment]
I understand the location of the group is not defined by the current location (goto)? How is the location specified?

**Comment by jrobinso** [comment]
It should be this, have you tried it?

```
group base_at_pos chr1:123456
```

**Comment by BuysDB** [comment]
Thanks for your reply. I did try it before and now double checked, and it does not work. It does enable the grouping option, but for the last location I performed it manually and not for the location specified after the `group base_at_pos`

### Example:
I manually go to a location and perform the group by base:
![image](http://buysdb.nl/persistent/github/igv_grouping/manual_group.png)
Then I disable the grouping again.

Now I want to group by the next base at `chr1:12346882`
Script:
```
goto chr1:12346882
group base_at_pos chr1:12346882
```

Results in:
![image](http://buysdb.nl/persistent/github/igv_grouping/grouped_by_script.png)

The `group_by_base` does get enabled again, but for the previous location.

**Comment by jrobinso** [comment]
OK, sorry that wasn't clear.  So this is a bug.

**Comment by jrobinso** [comment]
Sorry for the long delay, this should be fixed now in version 2.8.4.   Re-open if you continue to have problems.
